### ISSUE TITLE
Missing attributes and incorrect keyboard controls on tab control

### ISSUE THREAD

**Issue by Decrepidos** [govau/design-system-components#674]
Hi. 
Great to see a Government department dedicated to inclusion.

I found two issues with the tab control. You are missing aria-selected on the selected tab. A tab control should be one tab stop. arrow keys should move focus between the tabs.
WAI-ARIA Authoring Practices 1.1 3.22 Tabs
https://www.w3.org/TR/wai-aria-practices/#tabpanel

The tab control can be found on the Accordions example page and has the tab heading HTML and REACT.
https://designsystem.gov.au/components/accordion/

As an aside the accordion trigger used in the side menu should be a button or at the very least the < a > should have role of button. Code pasted below. This is announced by screen readers as 'link expanded', which may cause confusion.
`<a href="#accordion-Released" class="au-accordion__title js-accordion au-accordion--open" aria-controls="accordion-Released" aria-expanded="true">Released <span class="badge badge--Released">26</span></a>`

Regards
Laurence

**Comment by sukhrajghuman** [comment]
hey @Decrepidos thanks so much picking this up!

You are 100% correct on everything you've said. We will start working on this shortly.
